test/ex_doc/formatter/html/autolink_test.exs

```diff
@@ -636,14 +636,31 @@ defmodule ExDoc.Formatter.HTML.AutolinkTest do
                "[`<<>>/1`](#{@elixir_docs}elixir/Kernel.SpecialForms.html#%3C%3C%3E%3E/1)"
     end
 
-    test "removes optional Elixir namespace" do
-      assert project_doc("`Elixir.Enum`", %{}) == "[`Enum`](#{@elixir_docs}elixir/Enum.html)"
+    test "deals with optional Elixir namespace" do
+      assert project_doc("`Elixir.Enum`", %{}) ==
+               "[`Elixir.Enum`](#{@elixir_docs}elixir/Enum.html)"
 
       assert project_doc("`Elixir.Enum.concat/1`", %{}) ==
-               "[`Enum.concat/1`](#{@elixir_docs}elixir/Enum.html#concat/1)"
+               "[`Elixir.Enum.concat/1`](#{@elixir_docs}elixir/Enum.html#concat/1)"
 
       assert project_doc("`t:Elixir.Enum.t/0`", %{}) ==
-               "[`Enum.t/0`](#{@elixir_docs}elixir/Enum.html#t:t/0)"
+               "[`Elixir.Enum.t/0`](#{@elixir_docs}elixir/Enum.html#t:t/0)"
+
+      assert project_doc(
```

Comment:
what does this assertion and the one below test that we don't already have covered?

Reply:
Yes, those assertions can be removed. :)

Reply:
testing this change
https://github.com/elixir-lang/ex_doc/pull/1044/files#diff-4cca810c5bca54191f6de3e9b8edafb0L557

Reply:
This case is tested on the assertion above. Still, if there are new cases necessary, I would add simple assertions like the ones above. :)